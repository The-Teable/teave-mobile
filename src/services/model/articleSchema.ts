/**
 * 백엔드와 협의가 필요한 스키마
 */
export interface Article {
  id: string;
  title: string;
  author: string;
  create_date: string;
  content: string;
}

export type ReviewProps = Article & { image_url: string };

export type QuestionProps = Article & { isCompleted: boolean };
