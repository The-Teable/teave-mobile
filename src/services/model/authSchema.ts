import { User } from "./userSchema";

export type LoginProps = Pick<User, "user_id"> & { password: string };

export type SignupPropsPartail = Pick<User, "birth" | "gender">;

export type SignupProps = Pick<User, "user_id" | "password" | "name" | "tel"> &
  Partial<SignupPropsPartail>;

export type LogoutProps = { refresh: string };
