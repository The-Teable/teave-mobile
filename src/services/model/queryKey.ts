export const QUERY_KEY = {
  GET_THEME_FILTERING: "GET_THEME_FILTERING",
};
