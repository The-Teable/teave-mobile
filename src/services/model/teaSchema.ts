export interface Tea {
  id: number;
  name: string;
  flavor: string;
  efficacies: string;
  site_url: string;
  image_url: string;
  create_date: Date;
  update_date: Date;
  sell_count: number;
  price: number;
  brand: string;
  type: string;
  theme: string;
}

export type ClickProductProps = Pick<Tea, "id">;

export type WishProductProps = Pick<Tea, "id">;

export type MainFilteringResults = {
  id: number;
  name: string;
  brand: string;
  type: string;
  flavor: string;
  caffeine: string;
  efficacies: string;
  price: number;
  description: string;
  image_url: string;
}[];

export type ThemeFiltering = {
  theme: string;
  tea_info: {
    id: number;
    name: string;
    brand: string;
    type: string;
    flavor: string;
    caffeine: string;
    efficacies: string;
    price: number;
    image_url: string;
  }[];
}[];
