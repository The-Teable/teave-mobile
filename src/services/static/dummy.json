{
  "promotionBanners": [
    { "url": "/image/promotion1.png", "href": "/" },
    { "url": "/image/promotion2.png", "href": "/" },
    { "url": "/image/promotion3.png", "href": "/" },
    { "url": "/image/promotion1.png", "href": "/" },
    { "url": "/image/promotion2.png", "href": "/" },
    { "url": "/image/promotion3.png", "href": "/" }
  ],
  "shopFilterdTeas": [
    {
      "id": 0,
      "url": "/image/goods3.svg",
      "brand": "알디프",
      "name": "카라멜 티메리카노",
      "price": 15000
    },
    {
      "id": 1,
      "url": "/image/goods1.svg",
      "brand": "알디프디프디프",
      "name": "카라멜 티메리카노카노카노",
      "price": 15000
    },
    {
      "id": 2,
      "url": "/image/goods2.svg",
      "brand": "알디프",
      "name": "카라멜 티메리카노",
      "price": 15000
    },
    {
      "id": 3,
      "url": "/image/goods3.svg",
      "brand": "알디프",
      "name": "카라멜 티메리카노",
      "price": 15000
    },
    {
      "id": 4,
      "url": "/image/goods1.svg",
      "brand": "알디프",
      "name": "카라멜 티메리카노",
      "price": 15000
    },
    {
      "id": 5,
      "url": "/image/goods2.svg",
      "brand": "알디프",
      "name": "카라멜 티메리카노",
      "price": 15000
    }
  ],
  "teaProductDetailInfo": {
    "tea_image_urls": ["/image/dummy_product1.png"],
    "brand_logo_url": "/image/dummy_tea_logo.png",
    "brand_name": "티퍼블릭",
    "tea_name": "카라멜 티메리카노",
    "price": 15000,
    "delivery_info": "3만원 이상 구매시 무료배송",
    "tea_detail": "/image/dummy_product_detail.svg",
    "reviews": [
      {
        "id": "0",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      },
      {
        "id": "1",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "ppap",
        "content": "굿"
      },
      {
        "id": "2",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      },
      {
        "id": "3",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      },
      {
        "id": "4",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      },
      {
        "id": "5",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      },
      {
        "id": "6",
        "title": "맛있어용",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "image_url": "",
        "content": "굿"
      }
    ],
    "questions": [
      {
        "id": "0",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      },
      {
        "id": "1",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": true,
        "content": "굿"
      },
      {
        "id": "2",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      },
      {
        "id": "3",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      },
      {
        "id": "4",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      },
      {
        "id": "5",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      },
      {
        "id": "6",
        "title": "먹는방법",
        "author": "teave123",
        "create_date": "2019-07-04T13:33:03.969Z",
        "isCompleted": false,
        "content": "굿"
      }
    ]
  }
}
