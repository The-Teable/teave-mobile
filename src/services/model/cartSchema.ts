import { Tea } from "./teaSchema";

export type CartProductProps = Pick<
  Tea,
  "id" | "brand" | "name" | "price" | "image_url"
> & {
  count: number;
  is_selected: boolean;
};

export type Delivery = {
  brand: string;
  price: number;
};
