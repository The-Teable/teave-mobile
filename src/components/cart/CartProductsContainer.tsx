import styled from "styled-components";
import { ProductInCart } from "../../services/model/cartSchema";
import { color } from "../../styles/palette";
import { Fragment } from "react";
import CartProduct from "./CartProduct";

const TEMP_DELIVERY_FEE = 3000;

const CartProductsContainer = ({
  products,
}: {
  products: Record<string, ProductInCart[]>;
}) => {
  return (
    <>
      {Object.entries(products).map(([brandName, brandProducts], i) => {
        const totalPrice = brandProducts.reduce(
          (acc, cur) => acc + (cur.is_selected ? cur.price * cur.count : 0),
          0
        );
        return (
          <Fragment key={i}>
            <S.BrandTitle>{brandName} 배송</S.BrandTitle>
            {brandProducts.map((brandProduct) => (
              <CartProduct brandProduct={brandProduct} />
            ))}
            {totalPrice > 0 && (
              <S.TotalPriceContainer>
                <S.DeliveryCost>
                  {totalPrice.toLocaleString()}원 + 배송비{" "}
                  {TEMP_DELIVERY_FEE.toLocaleString()}원 =
                </S.DeliveryCost>
                <S.TotalPrice>
                  합계 {(totalPrice + TEMP_DELIVERY_FEE).toLocaleString()}원
                </S.TotalPrice>
              </S.TotalPriceContainer>
            )}
          </Fragment>
        );
      })}
    </>
  );
};

export default CartProductsContainer;

const S: any = {};

S.BrandTitle = styled.div`
  display: flex;
  font-weight: 500;
  background-color: #ffffff;
  align-items: center;
  justify-content: center;
  height: 5rem;
  border-bottom: 1px solid ${color.gray200};
  margin-top: 1rem;
`;

S.TotalPriceContainer = styled.div`
  display: flex;
  background-color: #ffffff;
  align-items: center;
  justify-content: center;
  height: 5rem;
  border-top: 1px solid ${color.gray200};
`;

S.DeliveryCost = styled.div`
  font-size: 1.2rem;
  color: #808080;
  margin-right: 0.5rem;
`;

S.TotalPrice = styled.div`
  font-weight: bold;
`;
