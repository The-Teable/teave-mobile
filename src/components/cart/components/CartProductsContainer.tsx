import styled from "styled-components";
import { CartProductProps } from "../../../services/model/cartSchema";
import { color } from "../../../styles/palette";
import { Fragment } from "react";
import CartProduct from "./CartProduct";

const TEMP_DELIVERY_FEE = 3000;

const CartProductsContainer = ({
  products,
}: {
  products: Record<string, CartProductProps[]>;
}) => {
  return (
    <>
      {Object.entries(products).map(([brandName, brandProducts], i) => {
        const totalPrice = brandProducts.reduce(
          (acc, cur) => acc + (cur.is_selected ? cur.price * cur.count : 0),
          0
        );
        return (
          <Fragment key={i}>
            <StyledBrandTitle>{brandName} 배송</StyledBrandTitle>
            {brandProducts.map((brandProduct) => (
              <CartProduct brandProduct={brandProduct} key={brandProduct.id} />
            ))}
            {totalPrice > 0 && (
              <StyledTotalPriceContainer>
                <StyledDeliveryCost>
                  {totalPrice.toLocaleString()}원 + 배송비{" "}
                  {TEMP_DELIVERY_FEE.toLocaleString()}원 =
                </StyledDeliveryCost>
                <StyledTotalPrice>
                  합계 {(totalPrice + TEMP_DELIVERY_FEE).toLocaleString()}원
                </StyledTotalPrice>
              </StyledTotalPriceContainer>
            )}
          </Fragment>
        );
      })}
    </>
  );
};

export default CartProductsContainer;

const StyledBrandTitle = styled.div`
  display: flex;
  font-weight: 500;
  background-color: #ffffff;
  align-items: center;
  justify-content: center;
  height: 5rem;
  border-bottom: 1px solid ${color.gray200};
  margin-top: 1rem;
`;

const StyledTotalPriceContainer = styled.div`
  display: flex;
  background-color: #ffffff;
  align-items: center;
  justify-content: center;
  height: 5rem;
  border-top: 1px solid ${color.gray200};
`;

const StyledDeliveryCost = styled.div`
  font-size: 1.2rem;
  color: #808080;
  margin-right: 0.5rem;
`;

const StyledTotalPrice = styled.div`
  font-weight: bold;
`;
