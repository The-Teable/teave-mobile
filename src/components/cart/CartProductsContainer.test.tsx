import CartProductsContainer from "./CartProductsContainer";
import { render } from "@testing-library/react";
import { productInCart } from "../../services/static/dummy.json";
import { ProductInCart } from "../../services/model/cartSchema";

describe("CartProductsContainer", () => {
  it("matches snapshot", () => {
    const dividedByBrand = productInCart.reduce<
      Record<string, ProductInCart[]>
    >((acc: Record<string, ProductInCart[]>, cur: ProductInCart) => {
      acc[cur.brand] = cur.brand in acc ? [...acc[cur.brand], cur] : [cur];
      return acc;
    }, {});

    const utils = render(<CartProductsContainer products={dividedByBrand} />);
    expect(utils.container).toMatchSnapshot();
  });
});
