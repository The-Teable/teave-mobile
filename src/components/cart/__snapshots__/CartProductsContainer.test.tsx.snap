// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartProductsContainer matches snapshot 1`] = `
<div>
  <div
    class="CartProductsContainer__BrandTitle-sc-lly8tb-0 ePPQiI"
  >
    보향다원
     배송
  </div>
  <div
    class="CartProduct__ItemContainer-sc-1ypdwhr-0 gKFICd"
  >
    <div
      class="ToggleSelector__OptionContainer-sc-1mowai4-0 vRFUu"
    >
      <input
        checked=""
        class="ToggleSelector__Checkbox-sc-1mowai4-1 iHTHMb"
        id="0"
        type="checkbox"
      />
      <label
        class="ToggleSelector__CustomCheckbox-sc-1mowai4-2 fHZZOm"
        for="0"
      />
    </div>
    <div
      class="CartProduct__ItemWrapper-sc-1ypdwhr-1 cfqyGE"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2775%27%20height=%2790%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="홍차아람"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="CartProduct__ContentWrapper-sc-1ypdwhr-2 kdYZFe"
      >
        <div
          class="CartProduct__ItemTitle-sc-1ypdwhr-3 OJTjV"
        >
          홍차아람
        </div>
        <div
          class="CartProduct__UnitPrice-sc-1ypdwhr-4 AYwcY"
        >
          25,000
          원
        </div>
        <div
          class="CartProduct__CountWrapper-sc-1ypdwhr-5 kZBgde"
        >
          <input
            class="CartProduct__Count-sc-1ypdwhr-6 gzLNkJ"
            type="number"
            value="1"
          />
          개
        </div>
      </div>
    </div>
    <div
      class="CartProduct__Price-sc-1ypdwhr-7 bedCUK"
    >
      25,000
      원
    </div>
    <span
      style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
    >
      <span
        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
      >
        <img
          alt=""
          aria-hidden="true"
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2710%27%20height=%2710%27/%3e"
          style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
        />
      </span>
      <img
        class="CartProduct__DeleteButton-sc-1ypdwhr-8 cEXrvk"
        data-nimg="intrinsic"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
      />
      <noscript />
    </span>
  </div>
  <div
    class="CartProduct__ItemContainer-sc-1ypdwhr-0 gKFICd"
  >
    <div
      class="ToggleSelector__OptionContainer-sc-1mowai4-0 vRFUu"
    >
      <input
        checked=""
        class="ToggleSelector__Checkbox-sc-1mowai4-1 iHTHMb"
        id="1"
        type="checkbox"
      />
      <label
        class="ToggleSelector__CustomCheckbox-sc-1mowai4-2 fHZZOm"
        for="1"
      />
    </div>
    <div
      class="CartProduct__ItemWrapper-sc-1ypdwhr-1 cfqyGE"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2775%27%20height=%2790%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="딸기 정원"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="CartProduct__ContentWrapper-sc-1ypdwhr-2 kdYZFe"
      >
        <div
          class="CartProduct__ItemTitle-sc-1ypdwhr-3 OJTjV"
        >
          딸기 정원
        </div>
        <div
          class="CartProduct__UnitPrice-sc-1ypdwhr-4 AYwcY"
        >
          15,000
          원
        </div>
        <div
          class="CartProduct__CountWrapper-sc-1ypdwhr-5 kZBgde"
        >
          <input
            class="CartProduct__Count-sc-1ypdwhr-6 gzLNkJ"
            type="number"
            value="2"
          />
          개
        </div>
      </div>
    </div>
    <div
      class="CartProduct__Price-sc-1ypdwhr-7 bedCUK"
    >
      30,000
      원
    </div>
    <span
      style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
    >
      <span
        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
      >
        <img
          alt=""
          aria-hidden="true"
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2710%27%20height=%2710%27/%3e"
          style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
        />
      </span>
      <img
        class="CartProduct__DeleteButton-sc-1ypdwhr-8 cEXrvk"
        data-nimg="intrinsic"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
      />
      <noscript />
    </span>
  </div>
  <div
    class="CartProductsContainer__TotalPriceContainer-sc-lly8tb-1 keSCHy"
  >
    <div
      class="CartProductsContainer__DeliveryCost-sc-lly8tb-2 dOtDHN"
    >
      55,000
      원 + 배송비
       
      3,000
      원 =
    </div>
    <div
      class="CartProductsContainer__TotalPrice-sc-lly8tb-3 jGoesP"
    >
      합계 
      58,000
      원
    </div>
  </div>
  <div
    class="CartProductsContainer__BrandTitle-sc-lly8tb-0 ePPQiI"
  >
    투티거스 호랑이 다과점
     배송
  </div>
  <div
    class="CartProduct__ItemContainer-sc-1ypdwhr-0 gKFICd"
  >
    <div
      class="ToggleSelector__OptionContainer-sc-1mowai4-0 vRFUu"
    >
      <input
        class="ToggleSelector__Checkbox-sc-1mowai4-1 iHTHMb"
        id="2"
        type="checkbox"
      />
      <label
        class="ToggleSelector__CustomCheckbox-sc-1mowai4-2 fHZZOm"
        for="2"
      />
    </div>
    <div
      class="CartProduct__ItemWrapper-sc-1ypdwhr-1 cfqyGE"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2775%27%20height=%2790%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="홍차아람"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="CartProduct__ContentWrapper-sc-1ypdwhr-2 kdYZFe"
      >
        <div
          class="CartProduct__ItemTitle-sc-1ypdwhr-3 OJTjV"
        >
          홍차아람
        </div>
        <div
          class="CartProduct__UnitPrice-sc-1ypdwhr-4 AYwcY"
        >
          15,000
          원
        </div>
        <div
          class="CartProduct__CountWrapper-sc-1ypdwhr-5 kZBgde"
        >
          <input
            class="CartProduct__Count-sc-1ypdwhr-6 gzLNkJ"
            type="number"
            value="1"
          />
          개
        </div>
      </div>
    </div>
    <div
      class="CartProduct__Price-sc-1ypdwhr-7 bedCUK"
    >
      15,000
      원
    </div>
    <span
      style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
    >
      <span
        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
      >
        <img
          alt=""
          aria-hidden="true"
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2710%27%20height=%2710%27/%3e"
          style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
        />
      </span>
      <img
        class="CartProduct__DeleteButton-sc-1ypdwhr-8 cEXrvk"
        data-nimg="intrinsic"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
      />
      <noscript />
    </span>
  </div>
</div>
`;
